src/Core/src/Primitives/WebProcessTerminated.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Microsoft.Maui
+{
+	public class WebProcessTerminated
```

Comment:
Should we add `Sender` like we have for `PlatformPointerEventArgs`?

Reply:
What are your thoughts on copying the `PointerEventArgs` and `PlatformPointerEventArgs` pattern more closely? The `PointerEventArgs` is an `EventArgs` and can live in the Controls layer, but `PlatformPointerEventArgs` is morfe just a POCO container. This could live in the Core and be apssed as the event. Then the args just exposes the platform as a property:


pseudo pointer code that I trimmed:

```cs
public class PointerEventArgs : EventArgs
{
	public PlatformPointerEventArgs? PlatformArgs { get; private set; }

	public PointerEventArgs()
	{
	}

	internal PointerEventArgs(PlatformPointerEventArgs? args = null)
	{
		_getPosition = getPosition;
		PlatformArgs = args;
	}
}
```

Reply:
Applied changes to align this changes with the `PointerEventArgs `and `PlatformPointerEventArgs `pattern.